### PR TITLE
Update main.yml for Certification issue

### PR DESCRIPTION
**Pull request by cerealcoders**
Line added to remove certification issue while downloading elasticsearch

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @souravray88! I've created [OSPR-737](https://openedx.atlassian.net/browse/OSPR-737) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams. 

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ('this must be merged by XX date', and why that is)
- partner information ('this is a course on edx.org')
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the Github pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

We can't start reviewing your pull request until you've submitted a [signed contributor agreement](http://open.edx.org/sites/default/files/wysiwyg/individual-contributor-agreement.pdf) or indicated your institutional affiliation and added yourself to the [AUTHORS](https://github.com/edx/configuration/blob/master/AUTHORS) file. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst) file for more information.

**Comment by sarina** [comment]
Hey @souravray88 welcome! I put your PR in the "waiting on author" state until you follow the two instructions the bot mentioned. Let me know if you have any questions.

File: playbooks/roles/elasticsearch/tasks/main.yml [hunk 1]
```diff
@@ -34,6 +34,7 @@
     url={{ elasticsearch_url }}
     dest=/var/tmp/{{ elasticsearch_file }}
     force=no
+    validate_certs=no // added to remove certification issue while downloading elasticsearch
```
Comment (feanil) [inline playbooks/roles/elasticsearch/tasks/main.yml:37]: This should not be needed with ansible 1.9 and newer unless you are connecting to a server that has bad certs.  Have you updated your requirements recently?

**Comment by sarina** [comment]
@souravray88 - sounds like this may be an issue with your version of Ansible. I suggest you look into getting help on the edx operations mailing list. See https://open.edx.org/resources/e-mail-lists

**Pull request closed without merging**